src/main/java/com/capitalone/dashboard/model/FortifyScanReport.java

```diff
@@ -0,0 +1,126 @@
+package com.capitalone.dashboard.model;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
Can we use CollectionUtils version 4?

Reply:
Oops!. Actually these dependencies are not needed. Just removed. Thanks!.